mappings/net/minecraft/client/render/VertexConsumerProvider.mapping

```diff
@@ -1,21 +1,32 @@
 CLASS net/minecraft/class_4597 net/minecraft/client/render/VertexConsumerProvider
 	CLASS class_4598 Immediate
-		FIELD field_20952 defaultBuilder Lnet/minecraft/class_287;
-		FIELD field_20953 layerBuilders Ljava/util/Map;
+		FIELD field_20952 fallbackBuffer Lnet/minecraft/class_287;
+		FIELD field_20953 layerBuffers Ljava/util/Map;
 		FIELD field_20954 currentLayer Ljava/util/Optional;
 		FIELD field_20955 activeConsumers Ljava/util/Set;
 		METHOD <init> (Lnet/minecraft/class_287;Ljava/util/Map;)V
-			ARG 1 defaultBuilder
-			ARG 2 layerBuilders
+			ARG 1 fallbackBuffer
+			ARG 2 layerBuffers
 		METHOD method_22993 draw ()V
 		METHOD method_22994 draw (Lnet/minecraft/class_1921;)V
 			ARG 1 layer
-		METHOD method_22995 getConsumer (Lnet/minecraft/class_1921;)Lnet/minecraft/class_287;
+		METHOD method_22995 getBufferInternal (Lnet/minecraft/class_1921;)Lnet/minecraft/class_287;
 			ARG 1 layer
+		METHOD method_23796 draw (III)V
+			ARG 1 cameraX
+			ARG 2 cameraY
+			ARG 3 cameraZ
+		METHOD method_23797 draw (Lnet/minecraft/class_1921;III)V
+			ARG 1 layer
+			ARG 2 cameraX
+			ARG 3 cameraY
+			ARG 4 cameraZ
 	METHOD getBuffer (Lnet/minecraft/class_1921;)Lnet/minecraft/class_4588;
 		ARG 1 layer
 	METHOD method_22991 immediate (Lnet/minecraft/class_287;)Lnet/minecraft/class_4597$class_4598;
 		ARG 0 builder
+		ARG 1 buffer
 	METHOD method_22992 immediate (Ljava/util/Map;Lnet/minecraft/class_287;)Lnet/minecraft/class_4597$class_4598;
 		ARG 0 layerBuilders
-		ARG 1 fallbackBuilder
+		ARG 1 layerBuffers
+		ARG 2 fallbackBuffer
```

Comment:
Did something go wrong here?

Reply:
The argument indices seem too have been shifted up by one but weren't updated in yarn... 🤔

Reply:
guess running dropping invalid mappings can help

Reply:
Manually fixed it. I guess dropping invalid mappings doesn't fix parameters?

Reply:
Also, this was caused by a Procyon bug. Static methods in interfaces are being marked `default` rather than `static`!